### ISSUE TITLE
Feature request - push to a different branch

### ISSUE THREAD

**Issue by cyberwombat** [fork-dev/Tracker#441]
In Tower I am able to push to a different branch - for Heroku for example I must always push to master `git push heroku testbranch:master`. It would be great to be able to do so in Fork. I don't see a way now?

**Comment by svoop** [comment]
@DanPristupov Would it be possible to add the possibility to push to a differently named remote branch which *does not yet* exist?

Context: I'm working on a project which enforces remote branches not to use hierarchies. However, to better organize my work, I'm using hierarchies locally, e.g. 

```
LOCAL                     REMOTE
feature/cool_stuff    ->  cool_stuff
bug/broken_thing      ->  broken_thing
```

When I start to work on a new branch, the corresponding remote branch does not yet exist and therefore can't be selected from the dropdown. I have to switch to CLI and push manually:

```
git push origin feature/cool_stuff:cool_stuff
```

**Comment by DanPristupov** [comment]
> Would it be possible to add the possibility to push to a differently named remote branch which does not yet exist?

That was implemented in 2.1.

I think we can close the issue now.

**Comment by svoop** [comment]
@DanPristupov Didn't see it since I'm working around it with a custom command, but definitely comes in handy from time to time! Big thanks!!
